pkg/col/coldata/bytes.go

```diff
@@ -262,7 +262,8 @@ func (b *Bytes) copyElements(srcElementsToCopy []element, src *Bytes, destIdx in
 			// append the new value to b.buffer.
 			*e = element{}
 			//gcassert:bce
-			e.setNonInlined(srcElementsToCopy[i].getNonInlined(src), b)
+			x := srcElementsToCopy[i]
+			e.setNonInlined(x.getNonInlined(src), b)
```

Comment:
Was this explicitly declared as a new variable because of looping/shadowing concerns?

Reply:
This was to split it over two lines to appease gcassert. I think either Go or gcassert got better at reporting these inlined bounds checks (`getNonInlined` has bounds checks that wouldn't be satisfied outside of the loop).

Reply:
Thank you!